java/src/apps/GuiLafConfigPane.java

```diff
@@ -97,7 +97,7 @@ void doClickSelection(JPanel panel) {
     void doVerticalToolBar(JPanel panel) {
         panel.setLayout(new FlowLayout());
         mouseEvent = new JCheckBox(ConfigBundle.getMessage("GUIButtonVerticalToolBar"));
-        mouseEvent.setSelected(InstanceManager.getDefault(GuiLafPreferencesManager.class).isVerticalToolBar());
+        mouseEvent.setSelected(InstanceManager.getDefault(GuiLafPreferencesManager.class).isToolBarVertical());
```

Comment:
That the code I wrote to add the vertical toolbar choice to the Preferences -> Display panel. It's no longer connected to anything in the LayoutEditor code so it can be removed… (done!). Good catch. Thanks.